media-collections/create-rename-delete-search-and-download-as-zip.md

```diff
@@ -0,0 +1,39 @@
+# Create, rename, delete, search & download media collections as zip
+
+You can create, rename, delete, search, and download media collections as zip files using ImageKit's dashboard. Learn more about permissions for these operations for restricted users [here](../access-control-and-permissions/README.md#media-collection-permission-levels).
+
+## Create new media collection
+
+Navigate into the media collections page by clicking on "Media collections" in the menu bar. Once you are here, click on the "New" button in the top right corner. This would open a popup in which you can enter the name of the media collection and click on "Submit." A new media collection will be created. A media collection's name should be unique and can consist of alphanumeric characters, underscores, and hyphens.
```

Comment:
> A media collection's name should be unique and...

A media collection's name should be unique across the organization account and...

Reply:
done